packages/react/src/profiler.tsx

```diff
@@ -57,11 +54,10 @@ class Profiler extends React.Component<ProfilerProps> {
       return;
     }
 
-    const activeTransaction = getActiveTransaction();
-    if (activeTransaction) {
-      // eslint-disable-next-line deprecation/deprecation
-      this._mountSpan = activeTransaction.startChild({
-        description: `<${name}>`,
+    const activeSpan = getActiveSpan();
+    if (activeSpan) {
+      this._mountSpan = startInactiveSpan({
```

Comment:
just thinking, do we need to check here if `_mountSpan` (same below for `_updateSpan`) is already set, to ensure we clean up the span? 🤔 or are we fine just overwriting the reference?

Reply:
Yup we should do a check - 934da39b709ee7165cef31308addf15d56a4d0c3